### PR TITLE
Spanish translation README.adoc file examples/rest-xml-json

### PR DESCRIPTION
**Pull request by jrxxjr**
In the path examples/rest-xml-json, translate to spanish the document README.adoc in the document README_es.adoc.

### CODE REVIEW

File: examples/rest-xml-json/README_es.adoc [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+== El servicio REST: @Path, @Produces, @Consumes
+
+Aquí vemos un bean que usa las anotaciones @Path, @Produces y @Consumes.
+La anotación @Path identifica la plantilla de ruta de URI a la que responde el recurso y se especifica en el nivel de clase de un recurso.
+El valor de la anotación @Path
+es una plantilla de ruta de URI parcial relativa al URI base del servidor en el que se implementa el recurso, la raíz de contexto del WAR y el patrón de URL al que
+responde el servlet auxiliar de Jersey.
+La anotación @Produces se utiliza para especificar los tipos de medios MIME o las representaciones que un recurso puede producir y enviar al cliente.
+Si @Produces se aplica en
+el nivel de clase, todos los métodos de un recurso pueden producir los tipos MIME especificados de forma predeterminada. Si se aplica a nivel de método, sobrescribir cualquier @Produces
```
Comment (cesarhernandezgt) [inline examples/rest-xml-json/README_es.adoc:20]: "Si se aplica a nivel de método, sobrescribir cualquier @Produces" puede cambiar a "Si se aplica a nivel de método, sobrescribe cualquier @Produces"

File: examples/rest-xml-json/README_es.adoc [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+----
+
+== Pruebas
+
+=== Prueba del servicio JAXRS
+
+La prueba usa el ApplicationComposer de OpenEJB para hacerlo trivial.
+
+La idea es primero activar los servicios jaxrs. Esto se hace usando
+Anotación @EnableServices.
```
Comment (cesarhernandezgt) [inline examples/rest-xml-json/README_es.adoc:63]: "Esto se hace usando
Anotación @EnableServices." 

puede cambiar a

"Esto se hace usando la 
anotación @EnableServices."

File: examples/rest-xml-json/README_es.adoc [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .post(new Request("Hi REST!"), Response.class).getValue();
+        assertEquals("hi rest!", message);
+    }
+}
+----
+
+#Running
+
+Ejecutar el ejemplo es bastante simple. En el directorio ``rest-xml-json''
```
Comment (cesarhernandezgt) [inline examples/rest-xml-json/README_es.adoc:141]: Initial quotes are not the same as final single quotes

**Review by cesarhernandezgt — CHANGES_REQUESTED** [review]
Thanks, much @jrxxjr for the translation.
I just suggested a couple of Spanish grammar changes.

File: examples/rest-xml-json/README_es.adoc (hunk 1 above)
Comment (jrxxjr) [reply examples/rest-xml-json/README_es.adoc]: The correction it is maked.

File: examples/rest-xml-json/README_es.adoc (hunk 2 above)
Comment (jrxxjr) [reply examples/rest-xml-json/README_es.adoc]: The correction it is maked.

File: examples/rest-xml-json/README_es.adoc (hunk 3 above)
Comment (jrxxjr) [reply examples/rest-xml-json/README_es.adoc]: The correction it is maked.

**Review by cesarhernandezgt — APPROVED** [review]

**Pull request merged**